### PR TITLE
make a self review to my previous project

### PR DESCRIPTION
**Pull request by MrOmachi**
In this project, I make a self-review of my previous portfolio project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+                <li>html</li>
+                <li>bootstrap</li>
+                <li>Ruby</li>
+              </ul>
+              <button type="button" class="btn">See Project</button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
```
Comment (MrOmachi) [inline index.html:192]: In the index.html file, Please I suggest Putting all your sections between nav and footer into the main container as it is a professional way of using the semantic tag.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,216 @@
...
+  border: none;
+}
+
+.btn:hover {
+  background-color: #7d73fc;
+}
+
+.btn:active {
+  background-color: #5e42d7;
+}
```
Comment (MrOmachi) [inline style.css:216]: In the style.css file, I suggest putting a default margin(* {padding: 0;  margin: 0;}) and padding format at the top of your code in other to take the browse default margin and padding from your code.

**Review by MrOmachi — COMMENTED** [review]
Great Job, kindly make the few changes attached in the lines of code.

**Review by youmari — APPROVED** [review]
# Project Approved :trophy: :tada: 🟢 

Hi @MrOmachi 

Your project is complete!  You have done it exceptionally well. it's time to merge it :100: 
Congratulations! 🎉 

### To Highlight :+1: 

:heavy_check_mark: No linters error.
:heavy_check_mark: Details PR title and description.
:heavy_check_mark: Good commit messages.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also connect with me on [slack](https://microverse-students.slack.com/team/U02FTTCCU2V)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**